src/StdChains.sol

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.2 <0.9.0;
+pragma experimental ABIEncoderV2;
+
+import "./Vm.sol";
+
+/// @dev To add a new chain:
+///   1. Add it into the `Chains` struct, named using CamelCase.
+///   2. Initialize it in the `stdChains` declaration.
+///   3. Add a corresponding `else if` line into the constructor, remembering to check for both
+///      underscore and hyphenated versions of the chain-name when appropriate.
+abstract contract StdChains {
```

Comment:
Should we export `StdChains` in `Components`?

Reply:
Maybe? `StdUtils is StdChains`, so it's technically exposed anyway. TBH I'm not totally clear on the use case for `Components.sol`, I think if you could update the README with an explanation of the new architecture that would be really helpful (both for me and future contributors)

Reply:
Users should import from `Components.sol`, but aren't required to. Devs must import from `Components.sol`.

If https://github.com/foundry-rs/forge-std/pull/195#discussion_r1008373663 sounds good, then we are going to export it.

Reply:
Yep agreed with that linked comment, works for me 👌